src/main/java/frc/robot/commands/MoveDistance.java

```diff
@@ -51,7 +51,7 @@ public boolean isFinished() {
     double[] position = this.drivetrain.getPosition();
     if (position[0] == position[1])
       currentPosition = position[0];
-    if (Math.abs(currentPosition - targetDistance) == allowedError) {
+    if (Math.abs(currentPosition - targetDistance) <= allowedError) {
       return true;
     } else {
       return false;
```

Comment:
Since that expression evaluates to true or false, you are essentially saying "if true, return true, else return false", which is redundant. Try this instead: `return Math.abs(currentPosition - targetDistance) <= allowedError;`.